tests/dummy/app/templates/map.hbs

```diff
@@ -18,6 +18,7 @@
     onShow=(action "onErrorMessageShow")
```

Comment:
This `ui-message` became unnecessary now, and should be removed with it's action handlers.